src/data/finset/lattice.lean

```diff
@@ -219,7 +220,8 @@ end
 
 /-! ### inf -/
 section inf
-variables [semilattice_inf_top α]
+-- TODO: define with just `[has_top α]` where some lemmas hold without requiring `[order_top α]`
```

Comment:
This is marked as a Todo here but not for the matching comment in the sup section